rbc/heavydb/remoteheavydb.py

```diff
@@ -744,6 +749,7 @@ def load_table_columnar(self, table_name, **columnar_data):
                         arr_nulls = [v is None for v in arr]
                         if True in arr_nulls:
                             # TODO: null support for text
+                            # TODO: null support for Geo Types
```

Comment:
This only applies to arrays<Geo*> types. Do we support Array with geo types?

Reply:
Not yet.